src/bp/sdk/botpress.d.ts

```diff
@@ -1937,7 +1937,7 @@ declare module 'botpress/sdk' {
      * from the user waiting in the queue.
      * @param event - Current event in the action context, used to identify the queue
      */
-    export function isIncomingQueueEmpty(event: IO.Event): boolean
+    export function isIncomingQueueEmpty(event: IO.IncomingEvent): boolean
```

Comment:
We shouldnt be breaking anyone here as the incoming queue contains `IncomingEvent`s